src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # BinSkim Release History
 
+## Unreleased
+
+* BUGFIX: Fix `KeyNotFoundException` when required properties are removed from the config file. [#565](https://github.com/microsoft/binskim/pull/565)
```

Comment:
>KeyNotFoundException [](http://example.com/codeflow?start=15&length=20)

Mention the rules that are problematic, both the rule id and friendly.

Reply:
updated.